rpa-manager/modules/ROOT/pages/settings-adminsettings-reassignassets.adoc

```diff
@@ -0,0 +1,86 @@
+= Reassigning RPA Assets
+
+If an RPA user or team is deleted or disabled in Anypoint Access Management, RPA assets that were assigned to this user or team are shown in the *Reassign Assets* tab in the *Admin Settings* view of the *Settings* module.
```

Comment:
```suggestion
If an RPA user or team is deleted or disabled in Anypoint Access Management, RPA assets that assigned to this user or team are shown in the *Reassign Assets* tab in the *Admin Settings* view of the *Settings* module.
```

Reply:
Would it work to say the *Reassign Assets* tab in *Settings > Reassign Assets*?

Can you make the second part of the sentence active? 
Maybe "...the Reassign Assets tab in...

Reply:
![image](https://github.com/mulesoft/docs-rpa/assets/104761429/c1439eaa-aeb0-41c6-b683-c40aebbd8fea)